recipe/gen_patch_json.py

```diff
@@ -714,6 +714,13 @@ def _gen_new_index(repodata, subdir):
         if any(dep.startswith("mpich >=3.4.1,") for dep in deps):
             _pin_looser(fn, record, "mpich", upper_bound="4.0")
 
+        # TBB 2021 (oneTBB 2021) is incompatible with previous releases.
+        if has_dep(record, "tbb") and record.get('timestamp', 0) < 1614285451000:
```

Comment:
I guess this timestamp should be updated once https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/124 is deployed?

Reply:
Oh, yes, did you mean deployment of https://github.com/conda-forge/tbb-feedstock/pull/78 ?

Reply:
I wonder if I should set some timestamp in future and then align it after deployment if it is done sooner

Reply:
Oops 😅 I meant to link https://github.com/conda-forge/tbb-feedstock/pull/79 with the view that any builds from now onwards should already have the correct pinning

Reply:
Got it, https://github.com/conda-forge/tbb-feedstock/pull/79 is merged, so I've updated the timestamp to Thu Feb 25 2021 13:51:50 UTC

Reply:
Perfect, can you post the output of running `show_diff.py`

Reply:
Also 1b of https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/1227#issuecomment-781493615 can be added in parallel to this PR if you have a moment.

Reply:
`python show_diff.py > tbb-patched-version.diff.log`: [tbb-patched-version.diff.log](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/files/6044331/tbb-patched-version.diff.log)

Regarding 1b: I don't fully understand the idea of pinning in conda-forge-pinning-feedstock, but I've just created https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1241 to start, feel free to comment, reject or update it.

Reply:
BTW I've updated 2021.a0 --> 2021.0.0a0 to align with conda-forge/tbb-feedstock#79